gen_host_status.yml

```diff
@@ -0,0 +1,37 @@
+- hosts: all
+  gather_facts: false
+  tasks:
+    - name: A debug msg all hosts will show except for skipped ones
+      debug:
+        msg: "Playing {{ ansible_host }}"
+      when: "'_skipped' not in ansible_host"
+
+    - name: Hosts haven't really changed, but we will say they have
+      debug:
+        msg: "I am a changed host."
+      changed_when: "'_changed' in ansible_host"
```

Comment:
I think you need `when: "'_skipped' not in ansible_host"`

Otherwise, the "skipped" hosts will have an "ok" task, which means they will not be truly skipped.